tensorboard/uploader/uploader.py

```diff
@@ -763,6 +763,7 @@ def _write_blob_request_iterator(self, blob_sequence_id, seq_index, blob):
                 crc32c=None,
                 finalize_object=finalize_object,
                 final_crc32c=None,
+                final_size=len(blob),
```

Comment:
Please test this? (Should be easy to modify existing tests.)

Reply:
I went pretty deep on investigating the mock call args, but it was getting pretty messy. Not seeing a simple approach.